src/app/customer/Customer.tsx

```diff
@@ -60,42 +64,72 @@ export interface WithCheckoutCustomerProps {
 
 export interface CustomerState {
     isEmailLoginFormOpen: boolean;
+    isReady: boolean;
     hasRequestedLoginEmail: boolean;
 }
 
 class Customer extends Component<CustomerProps & WithCheckoutCustomerProps, CustomerState> {
     state: CustomerState = {
         isEmailLoginFormOpen: false,
+        isReady: false,
         hasRequestedLoginEmail: false,
     };
 
     private draftEmail?: string;
 
-    componentDidMount(): void {
+    async componentDidMount(): Promise<void> {
         const {
-            onReady = noop,
+            initializeCustomer,
             email,
+            onReady = noop,
+            onUnhandledError = noop,
+            providerWithCustomCheckout,
         } = this.props;
 
         this.draftEmail = email;
 
+        try {
+            await initializeCustomer({ methodId: providerWithCustomCheckout });
```

Comment:
What happens if `providerWithCustomCheckout` is null, do you still need to call `initializeCustomer`?

Reply:
No I don't need to call `initializeCustomer` if `providerWithCustomCheckout` is null. 
If `providerWithCustomCheckout` is null happens it will be initialized with default customer flow, that just runs fallback and returns checkout state.

I didn't make check for that because of your recommendation in previous pull-request: https://github.com/bigcommerce/checkout-js/pull/619#discussion_r664257825

Reply:
Ah okay cool as long as it's handled within `initializeCustomer` it should be fine.